Backend/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.jpa.open-in-view = true
```

Comment:
Tak z ciekawości, czemu to włączamy? Z tego co czytałem to to jest domyślnie na true, a ludzie piszą że warto to właściwie zmienić na false :D Nie działało bez tego coś?
https://stackoverflow.com/questions/30549489/what-is-this-spring-jpa-open-in-view-true-property-in-spring-boot